dima/react-movies/src/components/moviesContainer/moviesContainer.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import MovieInterface from "../../interfaces/movieInterface";
+import Movie from "./movie";
+import ErrorBoundary from "../errorBoundary";
+import Loader from "../loader";
+
+import "./moviesContainer.css";
+
+interface MoviesContainerProps {
+  movies: MovieInterface[];
+  isMoviesExisted?: boolean;
+}
+
+const MoviesContainer = ({ movies, isMoviesExisted }: MoviesContainerProps) => {
```

Comment:
Logic is not really clear in your component. Hard to read. Make it easier. Just simply replace **isMoviesExisted** flag with **loading** and make corresponding check in the upper component. So it will have much clearer interface.